pages/runtime/runtime.md

```diff
@@ -36,17 +98,95 @@ var server = app.listen(80, function () {
 })
 ```
 
-## Persistent Storage		
+## Access to /dev
+
+In many projects you may need to control or have access to some external hardware via interfaces like GPIO, I2C or SPI. On resin.io your container application will automatically have access to `/dev` and these interfaces since the container is run in [**privileged** mode](https://docs.docker.com/engine/reference/commandline/run/#/full-container-capabilities-privileged). This means you should be able to use any hardware modules like you would in a vanilla linux environment.
+
+__Note:__ If you are not using one of the docker base images recommended in our [base images wiki][base-image-wiki-link], then its most likely you will need to handle the updating of `/dev` via [udev][udev-link] yourself. You can see an example of how our base images handle this [here](https://github.com/resin-io-library/base-images/blob/master/debian/armv7hf/jessie/entry.sh#L54).
+
+## Tips, Tricks and Troubleshooting
 
-If you want specific data or configurations to persist on the device through the update process, you will need to store them in `/data` . This is a special folder on the device file system which is essentially a [docker data `VOLUME`][docker-volume-link].
+### Writing to logs on the Dashboard
+
+In general anything written from the application to `stdout` and `stderr` should appear on the device's dashboard logs. Have a look at some of our [example projects][projects-github] on github to get an idea of how to do this.
```

Comment:
Why do you say `in general`? isn't it always forwarded or are you accounting for the bugs in logs? 

I'd just say: Anything written from the application to `stdout` and `stderr` will appear on the device's dashboard logs.